### DISCUSSION TITLE
Nextjs static export

### DISCUSSION THREAD

**Discussion by Paultje52** [ReactTooltip/react-tooltip#990] [Q&A]
Hey! I'm trying to use react-tooltip in my nextjs project. When using `next dev`, the tooltips work like expected. However, when building and exporting my project to a static page to host on cloudflare pages, the tooltips don't show up. Searching in the build, I can find the text I added in the tooltips. Does someone know how I can resolve this issue?

**Comment by gabrieljablonski** [comment]
Can you have a quick look at #972 and see if it's the same thing?

We're working right now on a way to have the package work without any issues with Next.js 13, but as a workaround you can try doing `swcMinify: false` on your `next.config.js` file.

If it's something else, I'll ask you to open a new issue with some more details.
